src/main/java/net/formio/validation/validators/AbstractValidator.java

```diff
@@ -48,6 +48,18 @@ public InterpolatedMessage warning(String elementName, String messageKey, Arg ..
 	public InterpolatedMessage info(String elementName, String messageKey, Arg ... args) {
 		return message(elementName, Severity.INFO, messageKey, null, args);
 	}
+
+	public InterpolatedMessage error(String elementName, String messageText, String messageKey, Arg ... args) {
```

Comment:
@balcovladimir Nechtělo by to přeci jen nějaké odlišení ve stylu [error|warning|info]WithText, aby se např. původní error se třemi parametry (elementName, messageKey a 1 arg) nepovažoval za nově přidaný error s elementName, messageText a messageKey?

Reply:
Jo to my nedošlo, že se tohle může stát. Sakra tak to ještě přejmenuji

Reply:
Udělal jsem teda nový pull request https://github.com/beranradek/formio/pull/23